content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -57,7 +57,7 @@ Install Istio and expose the control plane so that your virtual machine can acce
 1. Expose the control plane using the provided sample configuration.
 
     {{< text bash >}}
-    $ kubectl apply -f @samples/istiod-gateway/istiod-gateway.yaml@
+    $ kubectl apply -f @samples/multicluster/expose-istiod.yaml@
```

Comment:
This seems like a bad directory for a yaml file that is used for accessing istiod in many places, not just multcluster (VM integration, external control plane, primary cluster control plane).